### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -234,7 +240,51 @@ There is a particular "code smell" when working with containers: *if parent cont
...
+```
+export default ComponentName;
+```
+
+#### <a name="#react-components-files-location"></a>Files Location
+
+Component and container files should be located next to `.scss` files in `src/components` directory. 
+Documentation pages for components should be located in same directory in subdirectory called `pages`.
+
+#### <a name="(#react-components-tests"></a>Tests
```
Comment (kdzwinel) [inline CONTRIBUTING.md]: `(`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -234,7 +240,51 @@ There is a particular "code smell" when working with containers: *if parent cont
 
 > Containers SHOULD use "holes" to influence child blocks position, padding, offset.
 
+### <a name="react-components"></a>React Components
```
Comment (kdzwinel) [inline CONTRIBUTING.md]: #?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -234,7 +240,51 @@ There is a particular "code smell" when working with containers: *if parent cont
...
+
+Each component/container should have its own file.
+Component are dump and should be written in pure function form.
```
Comment (kdzwinel) [inline CONTRIBUTING.md]: `dumb`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -234,7 +240,51 @@ There is a particular "code smell" when working with containers: *if parent cont
 
 > Containers SHOULD use "holes" to influence child blocks position, padding, offset.
 
+### <a name="react-components"></a>React Components
+
+Each component/container should have its own file.
+Component are dump and should be written in pure function form. 
+Each component should exported as default module like:
```
Comment (kdzwinel) [inline CONTRIBUTING.md]: `should be`, `like so:`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -234,7 +240,51 @@ There is a particular "code smell" when working with containers: *if parent cont
...
+```
+
+#### <a name="#react-components-files-location"></a>Files Location
+
+Component and container files should be located next to `.scss` files in `src/components` directory. 
+Documentation pages for components should be located in same directory in subdirectory called `pages`.
+
+#### <a name="(#react-components-tests"></a>Tests
+
+Each component should be tests. We are using [jest](https://facebook.github.io/jest/) as a test runner/framework.
```
Comment (kdzwinel) [inline CONTRIBUTING.md]: `tested`

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -234,7 +240,51 @@ There is a particular "code smell" when working with containers: *if parent cont
...
+```
+
+Good:
+```
+import ButtonPrimary, {types as buttonTypes} from 'components/buttons/ButtonPrimary';
+```
+#### <a name="#react-components-options"></a>Components options
+
+Component options should be stored in const object.
+Options should have singular form (alignment not alignments) and capitalized names like:
```
Comment (kdzwinel) [inline CONTRIBUTING.md]: `"alignment" not "alignments"`

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -234,7 +240,51 @@ There is a particular "code smell" when working with containers: *if parent cont
...
 
+```
+const ALIGNMENT = {START: 'start', END: 'end'};
+```
+
+Each component should export it configuration options (if it have some).
```
Comment (kdzwinel) [inline CONTRIBUTING.md]: `its`

File: CONTRIBUTING.md (hunk 2 above)
Comment (aju) [inline CONTRIBUTING.md]: ?

File: CONTRIBUTING.md (hunk 4 above)
Comment (aju) [inline CONTRIBUTING.md]: `Each file should export default module like so:` -> good?

File: CONTRIBUTING.md (hunk 1 above)
Comment (aju) [inline CONTRIBUTING.md]: thx

File: CONTRIBUTING.md (hunk 5 above)
Comment (aju) [inline CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md (hunk 6 above)
Comment (aju) [inline CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md (hunk 7 above)
Comment (aju) [inline CONTRIBUTING.md]: 👍

File: CONTRIBUTING.md (hunk 2 above)
Comment (kdzwinel) [inline CONTRIBUTING.md]: missing #, all other links have it

**Pull request merged**